src/pages/index.js

```diff
@@ -143,40 +245,45 @@ profileButtonEdit.addEventListener("click", () => {
   resetValidation(formElement, inputList, config);
 });
 
+//close button listeners
 editModalCloseBtn.addEventListener("click", () => {
```

Comment:
**Could be improved**

You can make a universal handler for any close buttons.

It will look something like this:

```js
// Find all close buttons
const closeButtons = document.querySelectorAll('.modal__close');

closeButtons.forEach((button) => {
  // Find the closest popup only once
  const popup = button.closest('.modal');
  // Set the listener
  button.addEventListener('click', () => closePopup(popup));
});



```

That’s why we make universal css classes for similar elements: here it’s `modal__close` for any close button.

So now you can add 100 modals: their close buttons will be handled automatically.

Reply:
Thank you for your feedback and suggestions. I appreciate them along with some insights.